docs/schemas/core/IfcProductExtension/Types/IfcSpaceTypeEnum.md

```diff
@@ -19,10 +19,10 @@ A space dedication for use as a parking spot for vehicles, including access, suc
 Gross Floor Area - a specific kind of space for each building story that includes all net area and construction area (also the external envelop). Provision of such a specific space is often required by regulations.
 
 ### INTERNAL
-
+A space within a defined boundary or structure.
```

Comment:
The space would always have a boundary even if on the exterior. Do we mean "physical boundary".

I don't know how INTERNAL works with all the various facilities we have. Does it have a specific usage in terms of Bridges or Waterways? Or is this only intended for usage within buildings?